### ISSUE TITLE
Please add an example

### ISSUE THREAD

**Issue by pushpankar** [meereeum/lda2vec-tf#2]
It will make it easier to understand and use.

**Comment by diegoantognini** [comment]
Could be really helpful !

**Comment by kkrasnas** [comment]
Please do. Based on the issues created here the code doesn't work properly, so we need more insights.

**Comment by akshaydeeti1996** [comment]
yeah, it would be really helpful if anyone can provide on how to use it. For me it is showing a lot of errors and even after solving the errors, the kernel is always getting stopped, why?

**Comment by MovGP0** [comment]
even better: skip the example and just write some unit tests.

**Comment by mwunderlich** [comment]
Agreed. An plain example with builds an LDA2VEC model from a text file and outputs the topics would be really, really helpful here.
